%3a14/vararg.scm

```diff
@@ -0,0 +1,84 @@
+; Part of Scheme 48 1.9.  See file COPYING for notices and license.
```

Comment:
Isn't an OPT-LAMBDA variable arguments library part of the SRFI libraries already? Do we need a new implementation? I'm pretty sure Chez Scheme already has some stuff built in for this as well.